docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:18.04
```

Comment:
Any reason we should use ubuntu and miniconda? Using the various [`node`](https://hub.docker.com/_/node) images with `npm install -g yarn@^1.17` allows for smaller images with specific node and yarn versions pinned.

Reply:
just tried swapping the base image to `node:12` and installing `yarn` as per #1288 and the image size actually increased from 1.23GB to 1.53GB. Unless you're interested in trying `slim` or `alpine` packages...

Reply:
changed to `node:12-slim` (~1GB)